templates/js/custom-copilot-assistant-assistants-api/README.md.tpl

```diff
@@ -0,0 +1,123 @@
+{{/enableTestToolByDefault}}
+> - [Teams Toolkit Visual Studio Code Extension](https://aka.ms/teams-toolkit) version 5.0.0 and higher or [Teams Toolkit CLI](https://aka.ms/teamsfx-toolkit-cli)
+> - An account with [OpenAI](https://platform.openai.com/).
+>
+> **Note**
+>
+> The `AssistantsPlanner` in Teams AI Library is currently in preview version.
+
+### Create your own OpenAI Assistant
+
+Before running or debugging your bot, please follow these steps to setup your own [OpenAI Assistant](https://platform.openai.com/docs/assistants/overview).
+
+**If you haven't setup any Assistant yet**
+
+> This app template provides script `src/creator.js` to help create assistant. You can change the instructions and settings in the script to customize the assistant.
+> 
+> After creation, you can change and manage your assistants on [OpenAI](https://platform.openai.com/assistants).
+
+1. Open terminal and run command `npm install` to install all dependency packages
+   ```
+   > npm install
+   ```
+1. After `npm install` completed, run command `npm run assistant:create -- <your-openai-api-key>`
+   ```
+   > npm run assistant:create -- xxxxxx
+   ```
+1. The above command will output something like "*Created a new assistant with an ID of: **asst_xxx...***"
+1. Fill in both OpenAI API Key and the created Assistant ID into `env/.env.*.user`
+   ```
+   SECRET_OPENAI_API_KEY=<your-openai-api-key>
+   SECRET_OPENAI_ASSISTANT_ID=<your-openai-assistant-id>
+   ```
+
+**If you already have an Assistant created**
+
+1. Fill in both OpenAI API Key and the created Assistant ID into `env/.env.*.user`
+   ```
+   SECRET_OPENAI_API_KEY=<your-openai-api-key>
+   SECRET_OPENAI_ASSISTANT_ID=<your-openai-assistant-id>
+   ```
+
+### Run Teams Bot locally
+
+1. First, select the Teams Toolkit icon on the left in the VS Code toolbar.
+{{#enableTestToolByDefault}}
+1. Press F5 to start debugging which launches your app in Teams App Test Tool using a web browser. Select `Debug in Test Tool (Preview)`.
+1. You will receive a welcome message from the bot, or send any message to get a response.
+
+**Congratulations**! You are running an application that can now interact with users in Teams App Test Tool:
+
+![ai assistant bot in Teams App Test Tool](https://github.com/OfficeDev/TeamsFx/assets/15644078/90868166-115b-4394-a0d2-272bd985d0aa)
+{{/enableTestToolByDefault}}
+{{^enableTestToolByDefault}}
+1. In the Account section, sign in with your [Microsoft 365 account](https://docs.microsoft.com/microsoftteams/platform/toolkit/accounts) if you haven't already.
+1. Press F5 to start debugging which launches your app in Teams using a web browser. Select `Debug in Teams (Edge)` or `Debug in Teams (Chrome)`.
+1. When Teams launches in the browser, select the Add button in the dialog to install your app to Teams.
+1. You will receive a welcome message from the bot, or send any message to get a response.
```

Comment:
Do we have the logic for welcome message? Also, I think we may instruct developers how to trigger actions here.

Reply:
No welcome message. Good catch.
Yes, my plan is to refine all README afterwards together with the wiki.